pkg/kubelet/kubelet.go

```diff
@@ -2346,10 +2348,13 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 	// Check whether network is configured properly
 	networkConfigured := kl.doneNetworkConfigure()
 
+	selinuxRequired := capabilities.Get().EnableSELinuxIntegration
+	selinuxConfigured := (selinuxRequired && selinux.SelinuxEnabled()) || !selinuxRequired
```

Comment:
Btw, I would also call this `selinuxEnabled`.  Configured makes it sound like it has something to do with policy.